server/controllers/stock/index.js

```diff
@@ -132,7 +132,7 @@ async function createStock(req, res, next) {
     });
 
     const isExit = 0;
-    const postingParams = [db.bid(documentUuid), isExit, req.session.project.id, req.session.enterprise.currency_id];
+    const postingParams = [db.bid(documentUuid), isExit, req.session.project.id];
```

Comment:
We already know it.  Internally, we value the stock in the enterprise currency id and there isn't any option to change it.  Even if you purchase stock in a foreign currency, we do the internal conversion so that it shows up correctly on the balance sheet.

There isn't anywhere that a user can change the currency associated with stock that I am aware of.